src/main/java/com/example/voucher/controller/response/Response.java

```diff
@@ -0,0 +1,42 @@
+package com.example.voucher.controller.response;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Response<T> {
+    private List<T> dtoList;
+
+    public Response(List<T> dtoList) {
+        this.dtoList = dtoList;
+    }
+
+    public Response(T dto) {
+        this.dtoList = new ArrayList<>();
+        this.dtoList.add(dto);
```

Comment:
주웅님은 아래의 코드에 대해 어떻게 생각하시나요?
```suggestion
       return new Response(Arrays.asList(dto));
```

Reply:
2e113b10920e32f30657a4d729b4d641a778b819 커밋 피드백 반영 
- 두 문장을 하나의 문장으로 축약할 수 있다는 점에서 깔끔하다고 생각합니다.
- 좀 더 깔끔하게 코드를 작성할 수 있도록 생각하겠습니다.